docs/programme/area-owners.md

```diff
@@ -0,0 +1,150 @@
+# Area Owners
+
+In order to make sure that nothing falls through the cracks, and that work is
+not unduly piled onto a single person, we have a system of area owners. Ideally
+these area owners cover all parts of the programme. This list is not set in
+stone, and we expect it to change over time.
+
+It is wise to make sure that each area owner has a deputy, who can step in when
+they are unavailable, and can at least be aware of what is going on in their
+area.
+
+## Health and Safety
+
+Responsibilities:
+
+* Safety and compliance inspections
+* Risk assessments
+* Area wardens
+* Fire safety
+
+## Media / Press
+
+Responsibilities:
+
+* Photos
+* Organisation of group photo
+* Blog posts/announcements
+* Social media (incl. competitor social media)
+* Inviting guests
+* Arranging guest speaker
+* Swag
+* Competition booklet
+* Wristbands/lanyards
+* Reception
+* Media consent arrangements
+* Website (NB srcomp)
```

Comment:
What does "NB srcomp" mean here?

Reply:
Excellent question, this is a literal copy from the source. Will expand but: note that there are interactions with srcomp for the competition website, and srcomp is ostensibly within the remit of Game.

Reply:
https://github.com/srobo/runbook/pull/176/commits/d943e4b2faf4b11d1bd3312689a75a62833f836a